corehq/apps/app_manager/static/app_manager/js/form_designer.js

```diff
@@ -164,6 +164,7 @@ hqDefine("app_manager/js/form_designer.js", function() {
             editDetails.setUpdateCallbackFn(function (name) {
                 $('#formdesigner .fd-content-left .fd-head-text').text(name);
                 $('.variable-form_name').text(name);
+                document.title = name + ' - CommCare HQ';
```

Comment:
So I know this is more of a feature request, but would it be possible to make this a little more informative? 
I'm thinking:
- "{form name} - Form Settings - CommCare HQ"
- "{form name} - Edit Form - CommCareHQ"
- "{module name} - Module Settings - CommcareHQ"

Right now if you have both the settings of the form and the form builder open at the same time it is impossible to know which one is which. With HQ tab soup, this gets a little frustrating.